projects/aca-content/src/lib/services/content-management.service.ts

```diff
@@ -580,7 +581,7 @@ export class ContentManagementService {
           next: (newVersionUploaderData: NewVersionUploaderData) => {
             switch (newVersionUploaderData.action) {
               case NewVersionUploaderDataAction.refresh:
-                this.store.dispatch(new ReloadDocumentListAction());
+                this.store.dispatch(new RefreshPreviewAction(newVersionUploaderData.node));
```

Comment:
the old ReloadDocumentList is still needed I guess, as you can upload a new version by dragging and dropping a file on another file, without viewer